apps/openmw/mwmechanics/aiwander.hpp

```diff
@@ -151,6 +151,8 @@ namespace MWMechanics
             static const std::string sIdleSelectToGroupName[GroupIndex_MaxIdle - GroupIndex_MinIdle + 1];
 
             static int OffsetToPreventOvercrowding();
+
+            float fIdleChanceMultiplier;
```

Comment:
Not quite the same.  This is a global, not part of an individual actor's state information.  Every actor will have the exact same value.
If I could figure out how it properly initialize it at runtime, it would be a static.